newbank/server/NewBankClientHandler.java

```diff
@@ -221,4 +220,45 @@ public synchronized void processRequest(CustomerID customer, String request) {
 				break;
 		}
 	}
+	public synchronized void processEmployeeRequest(EmployeeID employee, String request) {
+
+		if (request.startsWith("NEWACCOUNT")){
+			String requestString = request;
+			String accountName = requestString.replace("NEWACCOUNT", "");
```

Comment:
I think we want to add a single whitespace character after NEWACCOUNT so that on input NEWACCOUNT x, we set `x` to accountName and not ` x`

Reply:
will still work and we can trim otherwise, though

Reply:
Good spot